worlds/dark_souls_3/__init__.py

```diff
@@ -44,82 +49,120 @@ class DarkSouls3World(World):
     """
 
     game: str = "Dark Souls III"
-    option_definitions = dark_souls_options
+    options: DarkSouls3Options
+    options_dataclass = DarkSouls3Options
     topology_present: bool = True
```

Comment:
I know this wasn't changed in the PR but I don't think this is important for DS as the regions are not shuffled so the paths should not need to be present in the spoiler log